### PR TITLE
Random key generating & pickle dump

### PR DESCRIPTION
**Pull request by DPS0340**

### CODE REVIEW

**Comment by DPS0340** [comment]
please review this.

**Comment by sourishsarmah** [comment]
New updates have been pushed, so update your branch and fix the merge conflicts

File: OpenGodsEye.sh [hunk 1]
```diff
@@ -1 +1,7 @@
-chmod +x god
\ No newline at end of file
+#!/bin/sh
+
+chmod +x god
+
+
+PATH=$PATH:$(pwd)
+echo "PATH=\$PATH:$(pwd)" >> $HOME/.bashrc
```
Comment (sourishsarmah) [inline OpenGodsEye.sh]:
```suggestion
echo "export PATH=\$PATH:$(pwd)" >> $HOME/.bashrc
```

**Comment by DPS0340** [comment]
it works properly, but not immediately. so add 'exec bash'. it reloads bash.

**Comment by DPS0340** [comment]
BTW, could you add a build test tool like travis CI? it checks error.

**Comment by sourishsarmah** [comment]
You have not updated your branch with the recent changes.
The pull request has some merge conflicts, so I can't verify it unless you solve those conflicts.
The conflicts are in -
* god
* pass.py           // has been renamed to encrypt.py

**Comment by DPS0340** [comment]
wait a min

**Comment by DPS0340** [comment]
incomplete.

**Comment by DPS0340** [comment]
sorry for my poor git skill. but i think it's completed for now.

**Comment by DPS0340** [comment]
@sourishsarmah would you check this?

**Comment by DPS0340** [comment]
need to mkdir if the .god folder not exists?

**Comment by DPS0340** [comment]
when make key.

**Comment by sourishsarmah** [comment]
No, we don't have to make .god dir. The program will make one on its own.

**Comment by sourishsarmah** [comment]
Your random key generation code is working. But I have to make some changes to it.
I will accept the PR when the changes are made.

Thanks for your contribution.

**Comment by DPS0340** [comment]
thanks :D

**Pull request merged**